troposphere/iotanalytics.py

```diff
@@ -0,0 +1,248 @@
+from . import AWSObject, AWSProperty, Tags
```

Comment:
Could you add a copyright and license comment to this file?

Reply:
sure, it's done.